libs/sdk-ui-ext/src/internal/utils/sort.ts

```diff
@@ -200,56 +203,132 @@ function validateAttributeSort(attributeSort: ISortItem, availableSort: IAvailab
 function validateMeasureSortLocators(sortLocators: ILocatorItem[], availableLocators: ILocatorItem[]) {
     return (
         sortLocators.length === availableLocators.length &&
-        sortLocators.every((sortLocator, index) => isEqual(sortLocator, availableLocators[index]))
+        sortLocators.every((sortLocator, locatorIndex) =>
+            isEqual(sortLocator, availableLocators[locatorIndex]),
+        )
     );
 }
 
-function validateMeasureSort(measureSort: ISortItem, availableSort: IAvailableSortsGroup) {
+function isValidMeasureSort(measureSort: IMeasureSortItem, availableSort: IAvailableSortsGroup) {
     return (
-        isMeasureSort(measureSort) &&
         availableSort &&
         !!availableSort.metricSorts?.find((availableMetricSort) =>
             validateMeasureSortLocators(availableMetricSort.locators, sortMeasureLocators(measureSort)),
         )
     );
 }
 
-function getSortsValidity(currentSort: ISortItem[], availableSorts: IAvailableSortsGroup[]): boolean[] {
-    return currentSort.map((sortItem, index) => {
-        if (isAttributeAreaSort(sortItem)) {
-            return validateAreaSort(sortItem, availableSorts[index]);
-        }
-        if (isAttributeSort(sortItem)) {
-            return validateAttributeSort(sortItem, availableSorts[index]);
+function findReusableSort(
+    currentSort: ISortItem[],
+    availableSortGroup: IAvailableSortsGroup,
+    groupIndex: number,
+    attributeValidationFce: (attributeSort: ISortItem, availableSort: IAvailableSortsGroup) => boolean,
```

Comment:
I'm not familiar with `Fce` abbreviation. Is it from `funkce`? :D How about changing it to `Predicate`?

Reply:
fixed